.github/workflows/dockerhub-cleanup.yml

```diff
@@ -0,0 +1,42 @@
+name: DockerHub Scheduled Cleanup
+
+on:
+  schedule:
+    - cron: '00 00 * * sun'
+  workflow_dispatch:
+
+jobs:
+  dockerhub-cleanup-inactive:
+    name: Cleanup inactive xaynet tags on Dockerhub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup hub-tool
+        env:
+          DHUSER: ${{ secrets.DOCKER_USERNAME }}
+          DHTOKEN: ${{ secrets.DOCKER_PASSWORD }}
+        run: |
+          export DEBIAN_FRONTEND="noninteractive"
+          sudo apt update
+          sudo apt install -y jq
+          LATEST=$(curl -s "https://api.github.com/repos/docker/hub-tool/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
+          wget https://github.com/docker/hub-tool/releases/download/${LATEST}/hub-tool-linux-amd64.tar.gz -O /tmp/hub-tool-linux-amd64.tar.gz
+          tar xzvf /tmp/hub-tool-linux-amd64.tar.gz --strip-components 1 -C /tmp hub-tool/hub-tool
+          mkdir -pv -m 700 ~/.docker
+          chmod -v 600 ~/.docker/config.json
+          echo -ne "ewogICJ1c2VybmFtZSI6ICJESFVTRVIiLAogICJwYXNzd29yZCI6ICJESFRPS0VOIgp9Cg==" | base64 -d > /tmp/auth.json
+          echo -ne "ewogICJhdXRocyI6IHsKICAgICJodWItdG9vbCI6IHsKICAgICAgImF1dGgiOiAiREhVU0VSVE9LRU4iCiAgICB9LAogICAgImh1Yi10b29sLXJlZnJlc2gtdG9rZW4iOiB7CiAgICAgICJhdXRoIjogIkRIVVNFUiIKICAgIH0sCiAgICAiaHViLXRvb2wtdG9rZW4iOiB7CiAgICAgICJhdXRoIjogIkRIVVNFUiIsCiAgICAgICJpZGVudGl0eXRva2VuIjogIkpXVFRPS0VOIgogICAgfQogIH0KfQoK" | base64 -d > ~/.docker/config.json
+          RUSERTOKEN=$(echo -ne "${DHUSER}:${DHTOKEN}" | base64 -w0)
+          RUSER=$(echo -ne "${DHUSER}:" | base64 -w0)
+          RTOKEN=$(echo -ne "${DHTOKEN}" | base64 -w0)
```

Comment:
I thought of that, we _can_, of course, but what do we gain? I mean, I checked on my private repo where I was running tests and because RTOKEN is defined from DHTOKEN (which itself comes from a secret), nothing is displayed on the logs from the runners. 
So I don't see any gain we'd have by having that one also be a secret. If/when we change the Docker Hub Token, we'd need to update two secrets, the DOCKERHUB_TOKEN and RTOKEN, otherwise the CI would break.